pkg/resources/common/psa-validation.go

```diff
@@ -23,9 +23,9 @@ var psaLabels = []string{
 
 // IsUpdatingPSAConfig will indicate whether or not the labels being passed in
 // are attempting to update PSA-related configuration.
-func IsUpdatingPSAConfig(oldLabels, newLabels map[string]string) bool {
+func IsUpdatingPSAConfig(old map[string]string, new map[string]string) bool {
```

Comment:
We want to keep the old version (oldLabels, newLabels) because the other one would fail linting check iirc.